static/src/javascripts/types/global.d.ts

```diff
@@ -112,6 +112,7 @@ interface Config {
 		};
 		timingEvents?: GoogleTimingEvent[];
 	};
+	stage: Stage;
```

Comment:
I've moved `stage` to `window.guardian.config` instead of `window.guardian.config.page`

This is the same place DCR puts it.

We added stage to the window object a month ago when we first started the Okta migration so we are most likely the only consumer of it at the moment and should be good to move.